README.md

```diff
@@ -402,7 +404,7 @@ Options:
   -u, --user-agent <USER_AGENT>
           User agent
 
-          [default: lychee/x.y.z]
+          [default: lychee/0.18.1]
```

Comment:
Can we change that backt to x.z.y? We use it as a wildcard pattern to avoid that the tests fail when we bump the version without any other changes.